programs/ssl/ssl_server2.c

```diff
@@ -130,7 +130,7 @@ int main(void)
 #define DFL_SNI                 NULL
 #define DFL_ALPN_STRING         NULL
 #define DFL_CURVES              NULL
-#define DFL_MAX_EARLY_DATA_SIZE 0
+#define DFL_MAX_EARLY_DATA_SIZE NULL
```

Comment:
The changes in this file are out of scope for this PR now. Please move them to another PR. And I would say it is the case for all the changes related to testing. Otherwise, instead of reverting more commits, as this PR is going to be quite small eventually, it is probably better to just to rewrite the history.

Reply:
`opt.max_early_data_size` is a string. So it should be NULL. 

I will move it to separate PR.

Reply:
I have rewritten the history.